tests/test_klaviyo_automatic_fields.py

```diff
@@ -0,0 +1,72 @@
+from tap_tester import menagerie
+
+from tap_tester import connections, runner
+from base import KlaviyoBaseTest
+
+class AutomaticFieldsTest(KlaviyoBaseTest):
+    """
+    Ensure running the tap with all streams selected and all fields deselected results in the replication of just the 
+    primary keys and replication keys (automatic fields).
+    """
+    
+    def name(self):
+        return "klaviyo_automatic_fields_test"
+    
+    def test_run(self):
+        """
+        • Verify we can deselect all fields except when inclusion=automatic, which is handled by base.py methods
+        • Verify that only the automatic fields are sent to the target.
+        • Verify that all replicated records have unique primary key values.
+        """
+        expected_streams = self.expected_streams()
+        
+        # We are not able to generate test data so skipping two streams(mark_as_spam, dropped_email)
+        expected_streams = expected_streams - {"mark_as_spam", "dropped_email"}
```

Comment:
This line isn't required you can do this in the above line only.

Reply:
Updated